### PR TITLE
fix: fix workflow template

### PR DESCRIPTION
**Pull request by sdh100shaun**
## Description

Update workflow template to use latest version and fix inputs 
Related issue: n/a

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: workflow-templates/ci.yaml [hunk 1]
```diff
@@ -21,8 +21,6 @@ jobs:
     outputs:
       short_sha: ${{ steps.sha.outputs.SHORT_SHA }}
       pretty_branch_name: ${{ steps.branch.outputs.NAME }}
-    env:
-      BRANCH_NAME: ${{ github.ref_name }}
```
Comment (JoshuaLicense) [inline workflow-templates/ci.yaml:25]: Is this intentional?

File: workflow-templates/ci.yaml [hunk 2]
```diff
@@ -31,44 +29,46 @@ jobs:
       - name: Set archive name output
         id: branch
         run: |
-          export PRETTY_BRANCH_NAME=$(tr '/' '-' <<< "$BRANCH_NAME")
+          export PRETTY_BRANCH_NAME=$(tr '/' '-' <<< ${{ github.ref_name }})
```
Comment (JoshuaLicense) [inline workflow-templates/ci.yaml:32]: ^

**Comment by AnnaDodson** [comment]
We'll be releasing version 3.2.0 soon so easiest to go straight to that version

**Comment by sdh100shaun** [comment]
> We'll be releasing version 3.2.0 soon so easiest to go straight to that version

let's get this merged and fix the release process 
we need stuff out there that is current

**Pull request merged**